week-4/implement/repeat.test.js

```diff
@@ -23,3 +23,42 @@
 // Given a target string str and a negative integer count,
 // When the repeat function is called with these inputs,
 // Then it should throw an error or return an appropriate error message, as negative counts are not valid.
+
+function repeatString(str, count) {
+  if (count < 0) {
+    throw Error("Negative counts are not valid..");
+  }
+  let index = 1;
+  let output = "";
+
+  while (index <= count) {
```

Comment:
Good work - pretty minimal implimention that does all that is asked. Won't repeat my moan about the index, but I would recommend using a for loop here - it's a little tidier but that is a minor complaint